src/commands/debug/dotnet.ts

```diff
@@ -0,0 +1,24 @@
+import { flags as oclifFlags } from '@oclif/command'
+
+import { CustomCommand } from '../../oclif/CustomCommand'
+import debugDotnet from '../../modules/debug/dotnet'
+
+export default class DotnetDebug extends CustomCommand {
+  static description = 'Debug for .NET applications'
+
+  static examples = ['vtex debug dotnet debugInst']
```

Comment:
I talked to @rafaelbac and maybe we wouldn't want to document `debugInst`

Reply:
Since the argument is required, how should we do that? Just don't add examples?

Reply:
Maybe it would be better to no add examples in this case then - `vtex debug dotnet` is not supposed to be used by users, but by IDEs - maybe in the future we could add a link to a help on how to setup the IDE to debug dotnet using this command, what do you thing @rafaelbac?

Reply:
For now, we don't need a description of the `debugInst` parameter.  We also don't need examples of how to use the command. 

What do you guys think about changing the command's description to indicate that it's for IDEs only?

Something like this:
```
Debug .NET applications (IDEs only)
```
We already have some [documentation about how to use it](https://github.com/vtex/dotnet-getting-started#debugging), but it's not public yet (only for internal use).

Once it's publicly available, we can add it to the description, as @tiagonapoli suggested.

Does that make sense?

Reply:
Yes! I think it's better that way